pkg/library/container/mountSources.go

```diff
@@ -14,9 +14,10 @@ package container
 
 import (
 	devworkspace "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
-	"github.com/devfile/devworkspace-operator/pkg/config"
-	"github.com/devfile/devworkspace-operator/pkg/library/constants"
 	corev1 "k8s.io/api/core/v1"
+
+	constants2 "github.com/devfile/devworkspace-operator/pkg/constants"
```

Comment:
we need more constants, I won't approve unless we have constants3 as well ;-D

I know, it's just refactoring tools._
```suggestion
	devfileConstants "github.com/devfile/devworkspace-operator/pkg/constants"
```

or any other shorter name which makes clear it's from devfile/api

Reply:
Yeah, that's the downside of IDE import magic :D